components/serializer.rst

```diff
@@ -511,6 +511,26 @@ There are several types of normalizers available:
 
     Objects are normalized to a map of property names to property values.
 
+::class:`Symfony\\Component\\Serializer\\Normalizer\\JsonSerializableNormalizer`
```

Comment:
The colon @theofidry is confused of comes from the double colon here (the directive is `:class:`).